IDE/Renesas/e2studio/RA6M3G/test-wolfcrypt/src/wolfssl_thread_entry.c

```diff
@@ -0,0 +1,11 @@
+#include <wolfssl/wolfcrypt/settings.h>

```

Comment:
This file needs wolfSSL copyright added to top.

Reply:
https://github.com/wolfSSL/wolfssl/blob/91a9117e1b7e7ca316299cef6dfc2243aa127651/IDE/Renesas/e2studio/RA6M3G/test-wolfcrypt/src/wolfssl_thread_entry.c#L1-L22

Thanks, updated.